src/box/index.cc

```diff
@@ -153,6 +154,16 @@ exact_key_validate(struct key_def *key_def, const char *key,
 	return key_validate_parts(key_def, key, part_count, false, &key_end);
 }
 
+int
+position_validate(const struct index_def *index_def, const char *key,
```

Comment:
Let's drop this method and call `exact_key_validate` directly.

Reply:
done